src/components/Callout/CalloutContent.tsx

```diff
@@ -130,6 +130,12 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
       this.props.onLayerMounted();
     }
 
+    // Set default directionalHint according to rtl setting.
+    // The is added here so that getRTL() is called on the client only.
+    if (!CalloutContent.defaultProps.directionalHint) {
```

Comment:
I don't think this is quite right. defaultProps static is used by React to provide default prop values into the constructor for initial rendering. You can't change them when the component is mounted... 

I'm not quite sure what the best approach here is; we have a default value because it's annoying to provide one. Instead, why don't we default to use "bottomAutoEdge". (I believe that's an option.)